src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitButton.razor.cs

```diff
@@ -8,25 +8,56 @@ public partial class BitButton
     {
         private ButtonStyle buttonStyle = ButtonStyle.Primary;
 
-
         private int? tabIndex;
 
+        /// <summary>
+        /// Allow focus on button when it is disabled
+        /// </summary>
         [Parameter] public bool AllowDisabledFocus { get; set; } = true;
+
+        /// <summary>
+        /// Define a string value to describe the button for a screen reader
+        /// </summary>
         [Parameter] public string? AriaDescription { get; set; }
+
+        /// <summary>
+        /// Hide button from a screen reader and display it visually
+        /// </summary>
         [Parameter] public bool AriaHidden { get; set; }
 
+        /// <summary>
+        /// Define a string to label the button for a screen reader
+        /// </summary>
         [Parameter] public string? AriaLabel { get; set; }
 
+        /// <summary>
+        /// If provided, button renders as an anchor
+        /// </summary>
         [Parameter] public string? Href { get; set; }
 
+        /// <summary>
+        /// If Href provided, specifies how to open the link, Possible values: _blank | _self | _parent | _top 
```

Comment:
there is no restriction for the value of this parameter, so remove the possible values and just mention the target of the anchor tag element